code/__HELPERS/matrices.dm

```diff
@@ -24,3 +24,40 @@
 		animate(transform = matrices[i], time = speed)
 		//doesn't have an object argument because this is "Stacking" with the animate call above
 		//3 billion% intentional
+
+
+//Dumps the matrix data in format a-f
+/matrix/proc/tolist()
+	. = list()
+	. += a
+	. += b
+	. += c
+	. += d
+	. += e
+	. += f
+
+//Dumps the matrix data in a matrix-grid format
+/*
+  a d 0
+  b e 0
+  c f 1

```

Comment:
well since they're in this format in the ref, the 0 0 1 are necessary to actually PRINT it in that format.
(```tolist()``` doesn't care about the grid format, hence why it skips out on 0,0,1)